tests/test_client/tests.py

```diff
@@ -202,6 +202,43 @@ def test_follow_relative_redirect_no_trailing_slash(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.request['PATH_INFO'], '/accounts/login/')
 
+    def test_follow_307_redirect(self):
+        """A URL with a 307 redirect"""
+        for method in ('get', 'post', 'head', 'options', 'put', 'patch', 'delete', 'trace'):
+            req_method = getattr(self.client, method)
+            response = req_method('/redirect_view_307/', data={'value': 'test'}, follow=True)
+            self.assertEqual(response.status_code, 200)
+            self.assertEqual(response.request['PATH_INFO'], '/post_view/')
+            self.assertEqual(response.request['REQUEST_METHOD'], method.upper())
+
+    def test_follow_308_redirect(self):
+        """A URL with a 307 redirect"""
```

Comment:
Despite the name, this method seems to be testing a 307, not a 308. Copy and paste error?

Reply:
It's been a while since I've looked at this, but no, confusingly the 307 view also returns 308. I'll split it out into two different methods.